docs/development/extensions-core/postgresql.md

```diff
@@ -27,6 +27,9 @@ To use this Apache Druid extension, [include](../../configuration/extensions.md#
 
 ## Setting up PostgreSQL
 
+To avoid issues with upgrades that require schema changes to a large metadata table, chose a PostgreSQL version that supports instant ADD COLUMN semantics.
+For example, any currently supported version of PostgreSQL, version 11 or later.
```

Comment:
Version 11 supports instant add where the default value of new column is not null. So far, I haven't found a scenario where we added a column with non-null default value. So we can get rid of this line.